packages/flutter/lib/src/material/slider_theme.dart

```diff
@@ -991,7 +1009,10 @@ class RoundSliderTickMarkShape extends SliderTickMarkShape {
     final Paint paint = Paint()..color = ColorTween(begin: begin, end: end).evaluate(enableAnimation);
 
     // The tick marks are tiny circles that are the same height as the track.
-    final double tickMarkRadius = sliderTheme.trackHeight / 2;
+    final double tickMarkRadius = getPreferredSize(
+        isEnabled: isEnabled,
+        sliderTheme: sliderTheme
+    ).width / 2;
```

Comment:
De-indent by 2 spaces and add trailing comma:

```
final double tickMarkRadius = getPreferredSize(
  isEnabled: isEnabled,
  sliderTheme: sliderTheme,
).width / 2;
```

Reply:
Done